packages/expo-gl/README.md

```diff
@@ -23,6 +23,9 @@ expo-gl requires that you also install and configure [expo-gl-cpp](https://githu
 npm install expo-gl
 ```
 
+### Compability
+expo-gl requires React Native version 0.57.x.
```

Comment:
The code on master should work with 0.59.x; this should be a table rather than just one sentence.

Reply:
What sort of information would you like to have in the table? A list of RN-versions with compatible expo-gl versions? If so, where can I find this information?

PS; I tried building the example in https://github.com/expo/expo/issues/4279 with the master branches of expo-gl and expo-gl-cpp. This results in a build error `No known instance method for selector 'executeUIBlock:forView:implementingProtocol:' at https://github.com/expo/expo/blob/16c1407f6ce0293e8d0beb383f4616a45c7bca6d/packages/expo-gl/ios/EXGL/EXGLObjectManager.m#L146

Working around this error, the app still crashes as described in https://github.com/expo/expo/issues/4279

Reply:
```suggestion

To use `expo-gl` with React Native 0.58.0 or newer you will need to use `5.x.x` version of `expo-gl` and at least `0.4.0` of `react-native-unimodules`. Here is the table showing compatibility between these three packages:

| expo-gl | react-native-unimodules | react-native |
| ------- | ----------------------- | ------------ |
| <=4.x.x | 0.3.x                   | <=0.57.x     |
| >=5.0.0 | >=0.4.0                 | *            |
```